<?xml version="1.0"?>
<launch>

  <param name="color_topic" value="/camera/color/image_raw"/>
  <param name="depth_topic" value="/camera/aligned_depth_to_color/image_raw"/>
  

</launch>